kubernetes/samples/scripts/create-weblogic-domain/manually-create-domain/domain.yaml

```diff
@@ -99,6 +99,21 @@ spec:
   clusters:
   - clusterName: cluster-1
     serverStartState: "RUNNING"
+    serverPod:
+      # Instructs Kubernetes scheduler to prefer nodes for new cluster members where there are not
+      # already members of the same cluster.
+      affinity:
+        podAntiAffinity:
```

Comment:
Do the MII sample tests pass with this change?  See the WLS instructions for running the stand-alone version in 'src/integration-tests/model-in-image/README'.  (I've only tested on OCI boxes - might not work directly on a mac.)    This will run the sample _and_ also run the 'check generated source' test I describe below in my other comment.